ic-os/components/monitoring/filebeat/filebeat.service

```diff
@@ -10,15 +10,17 @@ Wants=bootstrap-ic-node.service
 # We must wait for var to be mounted over before interacting with it
 After=var.mount
 Wants=var.mount
-# Only start Filebeat if configuration file exists
-ConditionPathExists=/boot/config/filebeat.conf
 
 [Service]
 User=filebeat
 Group=filebeat
 Environment="GODEBUG='madvdontneed=1'"
 ExecStartPre=+/opt/ic/bin/setup-filebeat-permissions.sh
-ExecStartPre=+/opt/ic/bin/generate-filebeat-config.sh -j /boot/config/filebeat.conf -i /etc/filebeat/filebeat.yml.template -o /run/ic-node/etc/filebeat/filebeat.yml
+ExecStartPre=+/opt/ic/bin/generate-filebeat-config.sh -i /etc/filebeat/filebeat.yml.template -o /run/ic-node/etc/filebeat/filebeat.yml
+
+# Only start Filebeat if configuration file is generated
+ExecCondition=/usr/bin/test -f /run/ic-node/etc/filebeat/filebeat.yml
+
```

Comment:
ConditionPathExists is replaced with a ExecCondition now that filebeat.conf no longer exists